alpha/lib/model/LiveEntry.php

```diff
@@ -448,8 +452,17 @@ protected function getMediaServersHostnames()
 	/**
 	 * @return boolean
 	 */
-	public function hasMediaServer($currentDcOnly = false)
+	public function isCurrentlyLive($currentDcOnly = false)
 	{
+		if ($this->getExplicitLive())
+		{
+			$isAdmin = kCurrentContext::$ks_object && kCurrentContext::$ks_object->isAdmin();
+			$userIsOwner = kCurrentContext::getCurrentKsKuserId() == $this->getKuserId();
+			$isUserAllowedPreview = $this->isEntitledKuserEdit(kCurrentContext::getCurrentKsKuserId());
```

Comment:
Why not use the field "$entitledUsersView" do we really want to couple between isEntitledKuserEdit to this feature?